src/main/java/xbot/common/injection/SimulationComponent.java

```diff
@@ -0,0 +1,11 @@
+package xbot.common.injection;
+
+import javax.inject.Singleton;
+
+import dagger.Component;
+
+@Singleton
+@Component(modules = DaggerSimulationModule.class)
+public abstract class SimulationComponent extends UnitTestComponent {
```

Comment:
❔ Obviously you don't have ready access to a roborio, but I was curious if you were able to measure start-up time locally via the simulator to see what if any change there was?

Reply:
Because Guice is still doing most of the work, I don't expect any significant impact yet. I've just swapped singletons with other singletons. More will need to be done before trying a comparison, in particular updating CommonLibFactory.